specification/indices/analyze/IndicesAnalyzeRequest.ts

```diff
@@ -59,6 +59,15 @@ export interface Request extends RequestBase {
      */
     index?: IndexName
   }
+  query_parameters: {
+    /**
+     * Index used to derive the analyzer.
+     * If specified, the `analyzer` or field parameter overrides this value.
+     * If no index is specified or the index does not have a default analyzer, the analyze API uses the standard analyzer.
+     * @doc_id analysis-standard-analyzer
+     */
+    index?: IndexName
+  }
```

Comment:
This index parameter is read in https://github.com/elastic/elasticsearch/blob/d9e751602dc4a874ef381797844f80044889fbef/server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestAnalyzeAction.java#L49 and has been there at least since 7.3.